src/theme/components/hero.ts

```diff
@@ -255,11 +283,11 @@ const fiftyFifty = {
       lg: "50%",
     },
   },
-};
-const Hero = {
-  baseStyle: {
+});
+const Hero = defineMultiStyleConfig({
+  baseStyle: definePartsStyle({
```

Comment:
Didn't realize this before. Was this always an empty object? You can remove the new lines and the `// ...`.

Reply:
Yes, it was always empty.